src/fmt/writer/termcolor/extern_impl.rs

```diff
@@ -479,6 +480,7 @@ impl Color {
             Color::White => Some(termcolor::Color::White),
             Color::Ansi256(value) => Some(termcolor::Color::Ansi256(value)),
             Color::Rgb(r, g, b) => Some(termcolor::Color::Rgb(r, g, b)),
+            _ => None,
```

Comment:
If we do bump the MSRV, let's also make this function return `Color` instead of `Option<Color>`.

Reply:
Great catch, I totally missed that :)